### PR TITLE
Update lockable_trait.rst for blocking lock

### PR DESCRIPTION
**Pull request by lpussacq**
The method lock() of LockableTrait accept 2 parameters, the first one is the name used to create the lock file, and the second is a value indicating if we should wait until the lock is release. Thus, the line $this->lock(true); should be $this->lock('someRandomLockName', true);

### CODE REVIEW

File: console/lockable_trait.rst [hunk 1]
```diff
@@ -30,7 +30,7 @@ that adds two convenient methods to lock and release commands::
             }
 
             // If you prefer to wait until the lock is released, use this:
-            // $this->lock(true);
+            // $this->lock('lockname', true);
```
Comment (xabbuh) [inline console/lockable_trait.rst]: What about passing  `null` as the first argument instead to use the default lock name?

Comment (lpussacq) [inline console/lockable_trait.rst]: ```null``` could do the trick as any string. Using ```null``` result in using the name define for the command. My point is to use the blocking functionnality of the trait, we need to add a first parameter before ```true```.

**Review by javiereguiluz — APPROVED** [review]
I've made the change requested by @xabbuh to move faster and make this mergeable.

By the way, now that we have the Lock component, I wonder if we should keep using this LockableTrait.

**Comment by xabbuh** [comment]
Thank you @lpussacq.

**Pull request merged**